crates/orchestrator/src/tests/jobs/da_job/mod.rs

```diff
@@ -84,32 +111,113 @@ async fn test_process_job() {
             nonces: vec![],
         },
     });
+
+    let pending_state_update = serde_json::to_value(&pending_state_update).unwrap();
+    let response = json!({ "id": 1,"jsonrpc":"2.0","result": pending_state_update });
+
+    let state_update_mock = server.mock(|when, then| {
+        when.path("/").body_contains("starknet_getStateUpdate");
+        then.status(200).body(serde_json::to_vec(&response).unwrap());
+    });
+
+    let response = DaJob
```

Comment:
rename this to avoid confusion with above variable

Reply:
Valid, implemented.